### PR TITLE
Say hello

### PR DESCRIPTION
**Pull request by dodecaphonix**

### CODE REVIEW

File: foo.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hello!
```
Comment (oloverbrown) [inline foo.txt:1]: Hey!

Comment (dodecaphonix) [reply foo.txt:1]: How do you like reviewing your first pull request?

Comment (oloverbrown) [reply foo.txt:1]: It's exhilarating!

Comment (dodecaphonix) [reply foo.txt:1]: lolol

Comment (dodecaphonix) [reply foo.txt:1]: See the green button that says "Review changes?"

**Review by oloverbrown — APPROVED** [review]

**Pull request merged**